kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -102,107 +181,38 @@ define([
                 diagnosis = Constants.DIAGNOSIS.VALID;
 
             return Promise.try(() => {
-                if (typeof value !== 'string') {
```

Comment:
This deletion block duplicates a call to `validateWorkspaceObjectNameString`

Reply:
😖